tests/acceptance/multiconfig-generic/base-log.err

```diff
@@ -0,0 +1 @@
+
```

Comment:
Whats this?

Reply:
I copied all these files from the baseline generic test and just changed it so I could test with multiple configs! So I believe this is just the error file to check against (empty because there should be no errors).